src/app/products/charges/create-charge/create-charge.component.html

```diff
@@ -140,19 +154,26 @@
                 Amount is <strong>required</strong>
               </mat-error>
               <mat-error *ngIf="chargeForm.controls.amount.hasError('pattern') && currencyDecimalPlaces !== undefined">
-                Amount must be <strong>a positive number with {{ currencyDecimalPlaces ? 'up to ' + currencyDecimalPlaces : 'no' }} decimal places</strong>
+                Amount must be <strong>a positive number with {{ currencyDecimalPlaces ? 'up to ' +
+                  currencyDecimalPlaces : 'no' }} decimal places</strong>
               </mat-error>
               <mat-error *ngIf="chargeForm.controls.amount.hasError('pattern') && currencyDecimalPlaces === undefined">
                 Amount must be <strong>a positive number</strong>
               </mat-error>
             </mat-form-field>
-
-            <mat-form-field fxFlex="48%">
+            <!--
+   * minCap and maxCap only allowed for loan and savings entitites
```

Comment:
Share charges can have mincap and maxcap 

Also your conditions are containing the share account cases as well. Please add them to the comment. 
Mincap and maxcap are allowed for some of the share charges too

Reply:
@PC-11-00 See above